### PR TITLE
Decreases time for PAI do use doorjack from 50 seconds to 20 seconds

### PR DESCRIPTION
**Pull request by V2LenKagamine**
## About The Pull Request

This small code change simply changes the amount of progress that is added per tick of door hacking time. As of current, each tick adds 4%, where this changes it to add 10% progress. All in all, this changes the time to hack a door from 50 seconds to 20 seconds.

## Why It's Good For The Game

As of Current version, hacking a door for a PAI takes 50 seconds to open one door. Doing so also alerts the AI that someone is hacking the door at the location of the door being opened. Even a somewhat experienced player is capable of fully hacking a door from scratch in under 30 seconds.(Assuming using Multitool with electric/no shock) On top of this, once the combination of wires is known, it is possible to open a door in <10 seconds. All of this without alerting the AI or any non looker to their location or doing

This pull request would make PAI's somewhat more valuable as a late-round tool, when materials may be scarce, and the only way forward is hacking a door. While a fully equipped player may take only a few seconds to open a door, a PAI would need to take nearly a full minute to open the door.
This change may encourage people to take PAI's as somewhat of a backup or a safety net. If you only need one door, a PAI might be faster than manual, but always slower than 2-3 doors.

Please discuss if this is too much of a buff.

**TL/DR: Doorjack WAAAAAY to slow.**
## Changelog
:cl:
balance: Decreased time to hack a door as PAI by 30 seconds.
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
Please change the title to accurately describe what your PR does and changes.
Recommended title: Speeds up pAI doorjacking by 150%.

**Comment by ATH1909** [comment]
> Please change the title to accurately describe what your PR does and changes.
> Recommended title: Speeds up pAI doorjacking by 150%.

*+150%

**Comment by trollbreeder** [comment]
> > Please change the title to accurately describe what your PR does and changes.
> > Recommended title: Speeds up pAI doorjacking by 150%.
> 
> *+150%

Corrected

**Comment by V2LenKagamine** [comment]
Oops. Forgot to properly name it, Had git troubles and didn't get to push request till it was really late for me.

**Comment by TheVekter** [comment]
I actually forgot the pAI could do this, because no one ever fucking uses it. Good change.

**Comment by OnlineGirlfriend** [comment]
I frequently use pAI doorjack. 50 -> 20 is an extreme buff and doesn’t give AI much time to respond to the jack. 

There are already several good reasons to have doorjack as a backup, namely scarcity of resources and avoidance of electric shock.

I also find a lot of people don’t even know about doorjack because pAI players aren’t encouraging its use. There are plenty of other uses for pAIs that can motivate people to pick them up and use them, so the onus is on the pAI to remind its master of their options / not use their entire memory immediately upon download. (Imo)

In short, I like this change, but 20 seconds is pretty fast, maybe too fast.

**Comment by RaveRadbury** [comment]
- We've already had metas in which pAI's are used as discount skeleton keys. Doorjack was balanced in such a way so that assistant's don't rush them for their door opening ability.
- This would make doorjack a "must download" app, which is a detriment to the versatility of the role as well as against the spirit of what is definitely meant to be an RP heavy role. We don't need any further erosion of RP on this codebase.
- Making a pAI on-par with a player's ability to hack a door, with the added benefit of not having to engage the wire UI _and_ being able to bypassed shocked doors is far too powerful.

This is incredible power creep. As the person who's been maintaining and developing pAI's this year (aside from the delightful holoform additions) I am very much against this PR.

If you want to reduce the time it takes to hack a door, my suggestion would be to add some form of interaction/minigame equivalent to the existing door hacking for players, so that a pAI could fail the minigame and hack in 50 seconds or do it skillfully and shave some time off. An example of this would be a Simon-like minigame where the expressed pattern takes a minimum amount of time to be played to the pAI and then the pAI can repeat the pattern back as quickly as they can. Of course, this would take more effort than **increasing a value by a factor of 250%**

**Comment by optimumtact** [comment]
no PAI's should be useful in other ways, not as a free set of tools for greytiding.
